rust-module-hello-world/rust.yml

```diff
@@ -20,7 +20,7 @@
           hello_world.msg == "Hello, World!"
   
   - name: Hello, Ansible!
-    action: rust_helloworld
+    rust_helloworld:
```

Comment:
This also requires adjusting args

Reply:
Right, I'm fixing removing the "args:" lines